lesson_9/dragla_marius/lesson9_classes.py

```diff
@@ -0,0 +1,64 @@
+# Build two classes of your choice that can model a real-life example. The class needs to meet the following requirements:
+#     at least 5 attributes each
+#     at least 2 methods each
+#     one class to inherit from another
+# As a demonstration create at least 5 instances of one class (preferably the child class) and call all the methods it holds
+# Ex: You can have one class (Country) that has general attributes about countries such as area, neighbours, cities etc and methods related to those attributes.
+# The second class can be a specific country (Romania) that has more specific attributes such as attractions, universities etc.
+
+class Products(object):
+
+    def __init__(self, category, brand, model, stock, price):
+        self.category = category
+        self.brand = brand
+        self.model = model
+        self.stock = stock
+        self.price = price
+
+    def category_check(self):
+        if self.category is 'Laptop':
+            return 'Category: Computers'
+        else:
+            return 'Category: Others'
+
+    def on_stock(self):
+        if self.stock > 0:
+            return 'Availability: On stock'
+        else:
+            return 'Availability: Out of stock'
+
+    def price(self):
+        return self.price
```

Comment:
Usually, the methods need to contain some form of functionality. If you just return the value the function call is similar to just reading the attribute from the instance.

Laptop.price will be the same as self.price. Also having the same name for methods and attributes is not good practice. Try to add some functionality to the method and change the name :smile:

Reply:
added functionality to convert price to euro